backend/scripts/import_doc_relationships/import_doc_relationships.py

```diff
@@ -0,0 +1,45 @@
+#!/usr/bin/env python3
+
+from datetime import datetime
+from app.db.models.document import DocumentRelationship, Relationship
+from app.db.session import SessionLocal
+
+
+def create_relationship_with_docs(db, name, document_ids):
+    print(f"Creating relationship '{name}' for ids: {document_ids}...")
+
+    relationship = Relationship(
+        name=name, type="SYSTEM", description=f"imported at {datetime.now()}"
```

Comment:
Would the description produced by this function not be more informative as something if built like this?

```suggestion
def create_relationship_with_docs(db, cclw_group_id, document_ids):
    name = f"imported_relation_{cclw_group_id}"
    print(f"Creating relationship '{name}' for ids: {document_ids}...")

    relationship = Relationship(
        name=name,
        type="SYSTEM",
        description=f"Group of related documents imported from CCLW ID {cclw_group_id}",
```

Reply:
I did have it like this originally - let me pop it back - I was over tidying :smile:

Reply:
Ah understood